tests/Controller/GalleryAdminControllerTest.php

```diff
@@ -48,6 +48,11 @@ class GalleryAdminControllerTest extends TestCase
 
     private $twig;
 
+    /**
+     * @psalm-suppress InternalMethod
+     *
+     * @see https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L64-L68
```

Comment:
We should create an issue/PR in Symfony for this.

Reply:
Also don't think they will mark them not internal just for testing purposes.

Reply:
I would say it worth the try. We might end with another way to test this code.

Reply:
in this case I guess the recommended way is using functional tests